lib/MusicBrainz/Server/Controller/Release.pm

```diff
@@ -94,20 +95,14 @@ after 'load' => sub {
     my $cdtoc_count = $c->model('MediumCDTOC')->find_count_by_release($release->id);
     $c->stash->{release_cdtoc_count} = $cdtoc_count;
 
-    # We need to load more artist credits in 'show'
-    if ($c->action->name ne 'show') {
-        $c->model('ArtistCredit')->load($release);
-    }
```

Comment:
This comment/condition was obsolete. We don't combine `$c->model('ArtistCredit')->load` calls in `show`, at least not since the change to load mediums with lots of discs asynchronously. It just had the exact same line as here, hence why I've removed it.